env/WCOSS2.env

```diff
@@ -126,10 +126,10 @@ elif [[ "${step}" = "ocnanalecen" ]]; then
     export NTHREADS_OCNANALECEN=${NTHREADSmax}
     export APRUN_OCNANALECEN="${APRUN_default} --cpus-per-task=${NTHREADS_OCNANALECEN}"
 
-elif [[ "${step}" = "marineanalletkf" ]]; then
+elif [[ "${step}" = "marineanlletkf" ]]; then
 
-    export NTHREADS_MARINEANALLETKF=${NTHREADSmax}
-    export APRUN_MARINEANALLETKF="${APRUN_default} --cpus-per-task=${NTHREADS_MARINEANALLETKF}"
+    export NTHREADS_MARINEANLLETKF=${NTHREADSmax}
+    export APRUN_MARINEANLLETKF="${APRUN_default} --cpus-per-task=${NTHREADS_MARINEANLLETKF}"
```

Comment:
This is incorrect specification for WCOSS2.

Reply:
@aerorahul Is there an example you could suggest to work off of? There seem to be a number of choices.

Reply:
Since JEDI does not yet employ threads, this could simply be:
```bash
export APRUN_MARINEANLLETKF="${APRUN_default}
```
When threading do come into play, 
```bash
export APRUN_MARINEANLLETKF="${APRUN_default} --ppn ${tasks_per_node}--cpu-bind depth --depth=${NTHREADS_MARINEANLLETKF}"
```
You could see the JEDI atm sections of this code for reference.